### ISSUE TITLE
Remove references to osalbool/boolean

### ISSUE THREAD

**Issue by skliper** [nasa/PSP#109]
The C99 "bool" and related true/false identifiers should be used instead.

This is related to [cfs_osal:#231 ]

**Comment by skliper** [comment]
_Imported from trac issue 105.  Created by jphickey on 2019-08-20T17:51:11, last modified: 2019-08-29T10:53:53_

**Comment by skliper** [comment]
_Trac comment by jphickey on 2019-08-20 17:55:14:_

Commit [changeset:67f3d1f] ready for review, branch `trac-105-use-c99-bool`

**Comment by skliper** [comment]
_Trac comment by jphickey on 2019-08-22 10:13:10:_

Note this is a dependency of [cfs_osal:#231 ] and must be merged before that one (or at the same cycle).

**Comment by skliper** [comment]
_Trac comment by jhageman on 2019-08-22 10:37:21:_

Approved for 'fast-track' to IC branch for test.  CCB will code review prior to merge to master.

**Comment by skliper** [comment]
_Trac comment by jphickey on 2019-08-22 16:11:27:_

Merged into branch `ic-ccb-20190821`

**Comment by skliper** [comment]
_Trac comment by jphickey on 2019-08-29 10:51:35:_

CCB approved for development merge on 2019-08-27

**Comment by skliper** [comment]
_Trac comment by jphickey on 2019-08-29 10:53:53:_

Merged to development branch
